services/graphql-server/src/graphql/query-builders/published-content.js

```diff
@@ -1,12 +1,22 @@
 const { getPublishedContentCriteria } = require('@base-cms/utils');
 
 module.exports = ({ query }, { input }) => {
-  const { since } = input;
+  const { since, after } = input;
+
+  if (since && after) {
+    const criteria = getPublishedContentCriteria({ since, after });
+    return { query: { ...query, ...criteria } };
+  }
 
   if (since) {
     const criteria = getPublishedContentCriteria({ since });
     return { query: { ...query, ...criteria } };
   }
 
+  if (after) {
+    const criteria = getPublishedContentCriteria({ after });
+    return { query: { ...query, ...criteria } };
+  }
+
```

Comment:
```suggestion
  if (since || after) {
    const criteria = getPublishedContentCriteria({ since, after });
    return { query: { ...query, ...criteria } };
  }
```

You don't need three ifs here, just send both arguments to `getPublishedCriteria`.